MvvmCross/iOS/iOS/Views/Presenters/Attributes/MvxRootPresentationAttribute.cs

```diff
@@ -3,6 +3,6 @@ namespace MvvmCross.iOS.Views.Presenters.Attributes
 {
     public class MvxRootPresentationAttribute : MvxBasePresentationAttribute
     {
-        public bool WrapInNavigationController { get; set; }
+		public bool WrapInNavigationController { get; set; } = true;
```

Comment:
Mhm but... the commit does not revert this change. `MvxRootPresentationAttribute` is the one that needs to be modified